.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  push:
+    tags:
+    - 'v*'  # Push events to matching v* (eg. v2.0.8)
```

Comment:
Is it possible to make it literally match `v*.*.*`? Or can we only use one wildcard?

Reply:
Indeed! Shall I?